src/index.js

```diff
@@ -29,7 +29,7 @@ const upgradeVersion = async () => {
   const req = deleteGunDB()
 
   // remove all local data so its not cached and user will re-login
-  await Promise.all([AsyncStorage.clear(), req.catch()])
+  await Promise.all([/*AsyncStorage.clear(), */ req.catch()])
```

Comment:
why was this removed?

Reply:
This AsyncStorage.clear() does not permit to login with mnemonic when isLoggedIn is true.

Reply:
this same .clear is in master and it doesn't have that problem. so probably the error is happening somewhere else and this shouldn't be removed

Reply:
As a result, we got that the user was redirected to the SignIn again.